packages/react-storage/src/components/StorageBrowser/context/controls/ActionSelect/ActionSelect.tsx

```diff
@@ -43,6 +43,13 @@ export function actionSelectReducer(
   state: ActionSelectState,
   _action: ActionSelectAction
 ): ActionSelectState {
+  if (_action.type === 'SELECT_ACTION_TYPE') {
+    // Update selected action with action passed from handleUpdateState
+    return { ...state, selected: _action };
```

Comment:
This is where we would update `items` parsed from the file picker selection, at least in the immediate term